src/lib/email.js

```diff
@@ -0,0 +1,27 @@
+import sendmail from 'sendmail';
+import StudentStore from '../store/student-store';
+
+const mailer = sendmail({
+    silent: true
+});
+
+export function sendMail(emails){
+    return new Promise((res, rej) => {
+        mailer({
+            from: 'testing@msoe.edu',
```

Comment:
NACK: Should this be something like `techsupport@msoe.edu`? We should also include the `replyTo` option with `'no-reply@msoe.edu'` or Jim's email address. https://github.com/guileen/node-sendmail#mail-options